src/main/java/net/fabricmc/fabric/api/client/model/fabric/VertexBuilder.java

```diff
@@ -49,7 +49,12 @@
      * Call to explicitly flush a completed quad. 
      * Useful in dynamic rendering contexts to flush the last quad.
      * Will be called implicitly when {@link #begin(ModelMaterial)} is
-     * called.
+     * called.<p>
+     * 
+     * TODO: consider making this required (not implicit) and returning
```

Comment:
Making this required is, IMO, an idea worth considering. (adding a default endBegin method not unlike Profilers would also be worth considering then) What's not a good idea - from the perspective of analysis/transformation/processing - would be variable-length strides per material.